python/pyspark/sql/readwriter.py

```diff
@@ -733,7 +733,7 @@ def save(self, path=None, format=None, mode=None, partitionBy=None, **options):
         :param partitionBy: names of partitioning columns
         :param options: all other string options
 
-        >>> df.write.mode('append').parquet(os.path.join(tempfile.mkdtemp(), 'data'))
+        >>> df.write.mode("overwrite").save("mydata.parquet")
```

Comment:
I think we should revert this change. Tmp directory will be created under target directory as of https://github.com/apache/spark/commit/a634d66ce767bd5e1d8553d1a2c32e2b1a80f642 ; however, now seems it will create under the current working directory.

Reply:
The change is really to use `.save`, as that what this is supposed to be an example of. If it makes sense to make a temp file to save to, yes I agree, leave that part the same.

Reply:
Oh, yes. We should use `.save`. Thanks for clarification.